src/Salt/UserBundle/Resources/views/Security/login.html.twig

```diff
@@ -31,7 +35,11 @@
                         <input type="hidden" name="_target_path" value="{{ redirect }}"/>
                     {% endif %}
 
-                    <button class="pull-right btn btn-primary" type="submit">Login</button>
+                    {% if google_captcha_site_key is not empty %}
+                        <button class="pull-right btn btn-primary btn-login g-recaptcha" data-sitekey="{{ google_captcha_site_key }}" data-callback="onSubmit">Login</button>
+                    {% else %}
+                        <button class="pull-right btn btn-primary btn-login" data-callback="onSubmit">Login</button>
```

Comment:
The regular submit button should probably remain a simple `type="submit"` button instead of having to call a javascript function (and wrap the js function below with the `if google...key ...` as well.